src/components/wizard/wizard.component.ts

```diff
@@ -178,27 +183,66 @@ export class WizardComponent implements AfterViewInit, OnDestroy {
      */
     next(): void {
 
+        this.currentStep = this.getCurrentStep();
```

Comment:
This doesn't really need to be a property of the class since we can call `getCurrentStep()` at any time. Let's change this to
```ts
const currentStep = this.getCurrentStep();
```

and remove the `currentStep: any;` above.